compiled_starters/cpp/.codecrafters/compile.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+#
+# This script is used to compile your program on CodeCrafters
+# 
+# This runs before .codecrafters/run.sh
+#
+# Learn more: codecrafters.io/program-interface
+
+# Exit early if any commands fail
+set -e
+
+cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE=${VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake
+cmake --build ./build
```

Comment:
Needs `exec` at the end - else signals sent to the shell script won't be forwarded to the child program